python-squid/src/main/java/org/sonar/python/tree/PythonTreeMaker.java

```diff
@@ -570,18 +567,22 @@ PyExpressionTree expression(AstNode astNode) {
     if (astNode.is(PythonGrammar.CALL_EXPR)) {
       return callExpression(astNode);
     }
+    if (astNode.getChildren().size() == 1) {
```

Comment:
Sorry, now that I see it, I think it should be: `if (astNode.is(PythonGrammar.EXPR) && astNode.getChildren().size() == 1)`. What do you think?